evap/evaluation/templates/base.html

```diff
@@ -85,41 +83,75 @@
+                }
+            }
+
+            applyTomSelect = function(elements, additionalOptions = {}) {
+                elements.forEach(function(element) {
+                    const minimumInputLength = element.options.length >= 50 ? 3 : 0;
+
+                    if(element.tomselect) {
+                        element.tomselect.destroy();
+                    }
+                    element.classList.remove("form-select");  // TomSelect applies their own matching classes / styles
+                    if(element.parentElement.classList.contains("is-invalid")) {  // hotfix for our wrongly placed is-invalid
+                        element.parentElement.classList.remove("is-invalid");
+                        element.classList.add("is-invalid");
+                    }
+                    const baseOptions = {
+                        createOnBlur: true,
+                        placeholder: "{% trans 'Please select...' %}",
+                        hidePlaceholder: true,
+                        minimumInputLength: minimumInputLength,
+                        render: {
+                            option_create: function( data, escape ) {return '<div class="create">' + escape(data.input) + '</div>';},
+                            no_results: function( data, escape ) {
+                                if(data.input.length < minimumInputLength) {
+                                    return `<div class="no-results">{% trans "Please enter ${minimumInputLength} characters or more..." %}</div>`;
+                                } else {
+                                    return '<div class="no-results">{% trans "No results found" %}</div>';
+                                }
+                            }
+                        },
+                        plugins: {},
+                    };
+                    if(element.hasAttribute("data-tomselect-fullwidth")) {
+                        // span needed to the "remove this icon" button is right-aligned
+                        baseOptions.render.item = function(data, escape) {return '<div class="w-100"><span class="w-100">' + escape(data.text) + '</span></div>';};
+                    }
+                    if(element.multiple) {
+                        baseOptions.plugins.clear_button = {"title": "{% trans 'Remove all items' %}"};
+                        baseOptions.plugins.remove_button = {"title": "{% trans 'Remove this item' %}"};
+                    }
+                    new MinimumInputLengthTomSelect(element, Object.assign({}, baseOptions, additionalOptions));
+                });
             };
-            applySelect2Disabled($('select[disabled]').not('.form-template select'));
+            applyTomSelect(document.querySelectorAll("select:not(.form-template select):not(.no-tomselect)"));
 
             // fix Bootstrap bug that selecting toggle buttons via keyboard controls is only visually
             // https://github.com/twbs/bootstrap/issues/26855
```

Comment:
Is this still needed? The issue is closed :thinking:

Reply:
Removed.